tests/django_restframework_gis_tests/test_filters.py

```diff
@@ -443,3 +481,25 @@ def test_DistanceToPointFilter_ValueError_distance(self):
         url_params = '?dist=wrong&point=12.0,42.0&format=json'
         response = self.client.get('%s%s' % (self.location_within_distance_of_point_list_url, url_params))
         self.assertEqual(response.data['detail'], 'Invalid distance string supplied for parameter dist')
+
+    @skipIf(not has_geometry_distance, 'Skipped test for Django < 3.0: missing feature "GeometryDistance"')
+    def test_DistanceToPointOrderingFilter_filtering_none(self):
+        url_params = '?point=&format=json'
+        response = self.client.get('%s%s' % (self.location_order_distance_to_point, url_params))
+        self.assertDictEqual(response.data, {'type': 'FeatureCollection', 'features': []})
+
+    @skipIf(not has_geometry_distance, 'Skipped test for Django < 3.0: missing feature "GeometryDistance"')
+    def test_DistanceToPointOrderingFilter_ordering_filter_field_none(self):
+        from .views import GeojsonLocationOrderDistanceToPointList as view
```

Comment:
is there a reason we have to import this here? Can we place the import at the top please? Or is there an issue if we do that?

Reply:
I just did it the same way as its done on `test_inBBOXFilter_filter_field_none`

Reply:
> I just did it the same way as its done on `test_inBBOXFilter_filter_field_none`

would be a good chance to fix that one as well, may have slipped from our attention